bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java

```diff
@@ -8426,7 +8433,7 @@ private void onMouseMove(final long eventTime, final int devXMouse, final int de
       if (_hoveredValuePointIndex != -1
 
             /*
-             * This reduces flickering when the slider is dragged, it's not prefect but better and
+             * This reduces flickering when the slider is dragged, it's not pefect but better and
```

Comment:
When fixing the wording then it should be fixed correctly

Reply:
Oups ! I'll fix it in my FR translation PR